### PR TITLE
[PRODX-24170]: added tests for SyncView component

### PR DESCRIPTION
**Pull request by AntonNos**
Reminder: Need to cover `handleAddCloud()` method
Screenshot 2022-07-20 at 14 23 21

UPD: this block wasn't covered after submitting. Should we cover it or we are good with this coverage percentage?

### CODE REVIEW

File: __mocks__/@k8slens/extensions.js [hunk 1]
```diff
@@ -264,15 +264,16 @@ Button.propTypes = {
...
   primary: propTypes.bool,
   waiting: propTypes.bool,
+  plain: propTypes.bool,
 };
 
-export const Icon = ({ interactive, smallest, ...props }) => {
-  return <i className={cx({ smallest, interactive })} {...props} />;
+const Input = ({ trim, ...props }) => {
+  const classes = [trim ? 'trim' : ''];
+  return <input className={classes.join()} {...props} />;
```
Comment (stefcameron) [inline __mocks__/@k8slens/extensions.js:272]: I think it'll be safer in the long term, as we add more classes for various scenarios, to use the `cx()` utility: `className={cx({ trim })}`

**Review by stefcameron — CHANGES_REQUESTED** [review]
Just one small change.

File: __mocks__/@k8slens/extensions.js (hunk 1 above)
Comment (AntonNos) [reply __mocks__/@k8slens/extensions.js]: Make sense. Fixed

**Review by stefcameron — APPROVED** [review]
Thank you!

**Comment by stefcameron** [comment]
Merging now since I need to modify this new SyncView.spec.js file to add more tests for some local changes.

**Pull request merged**